### PR TITLE
Fixezzz

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

**Comment by Davidherrmann** [comment]
S

**Comment by davoodinator** [comment]
https://github.com/RPCS3/rpcs3/pull/2782

breaks the fixes here

File: Utilities/StrFmt.cpp [hunk 1]
```diff
@@ -286,6 +286,15 @@ struct fmt::cfmt_src
 
 		return 0;
 	}
+
+	static constexpr std::size_t size_char  = 1;
+	static constexpr std::size_t size_short = 2;
+	static constexpr std::size_t size_int   = 0;
```
Comment (danilaml) [inline Utilities/StrFmt.cpp]: Why `0` (and why the duplication here and in `sys_libc_.cpp`? with same names but different values).

Comment (Nekotekina) [inline Utilities/StrFmt.cpp]: Special value. Different values describe host system and PS3 types.

Comment (danilaml) [inline Utilities/StrFmt.cpp]: Thanks, I understand better now. So `int` is only special on pc? Btw, char/short could be done with `sizeof` too for consistency.

**Pull request merged**